lib/core/DocsLayout.js

```diff
@@ -40,6 +40,17 @@ class DocsLayout extends React.Component {
     if (this.props.Doc) {
       DocComponent = this.props.Doc;
     }
+    const spawn = require('cross-spawn');
+    const timeSpan = spawn
+      .sync('git', [
+        'log',
+        '-1',
+        '--format=%ct',
+        `versioned_docs/${metadata.source}`,
```

Comment:
Can you refactor the function to `core/utils.js` and optionally write a unit test for it ? Try to make it as general as possible

It can be something like

```js
function (getGitLastUpdated) {
  const timeStamp = parseInt(spawn.sync('git', ['log', '-1', '--format=%ct', filepath]).stdout.toString('utf-8')) * 1000;
  return new Date(timeStamp).toLocaleString();
}
```